### PR TITLE
fix running error on taobao platform

### PR DESCRIPTION
**Pull request by bofeng-song**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by minggo** [comment]
Please describe the reason that causes the running error.

**Comment by bofeng-song** [comment]
> Please describe the reason that causes the running error.

The window.screencanvas property is undefined in isolated engine environments.

File: platforms/minigame/platforms/taobao-mini-game/wrapper/engine/AssetManager.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ downloader.register({
 });
 
 function loadImage (url, options, onComplete) {
-    const img = window.screencanvas.createImage();
+    const img = $global.screencanvas.createImage();
```
Comment (bofeng-song) [inline platforms/minigame/platforms/taobao-mini-game/wrapper/engine/AssetManager.js:33]: The window.screencanvas property is undefined in isolated engine environments.

Comment (minggo) [reply platforms/minigame/platforms/taobao-mini-game/wrapper/engine/AssetManager.js:33]: What does `isolated engine environments` mean?

Comment (bofeng-song) [reply platforms/minigame/platforms/taobao-mini-game/wrapper/engine/AssetManager.js:33]: The window.screencanvas property is undefined in separate engine plugin mode.

**Review by minggo — APPROVED** [review]

**Pull request merged**